### ISSUE TITLE
No devices found when looking for range lse6335f

### ISSUE THREAD

**Issue by SaintChimera** [ollo69/ha-smartthinq-sensors#295]
**Describe the bug**
Hello, first off thank you for developing this integration. I'm not sure whether to call this a feature or a bug, so i just picked one. I'm trying to get this set up with my range and after going through the setup, I am reaching the No SmartThinQ devices found error. I used a LG account from the start so no social media account issues should be related. I'm not sure if its not finding my range by a problem or if the range isn't supported but I'd love to get it working.

**Expected behavior**
Integration finds the range

**Environment details:**
 - Environment (HASSIO, Raspbian, etc): home assistant OS on raspberry pi 4
 - Home Assistant version installed: 2022.2.9
 - Component version installed: v0.12.10
 - Last know working version:
 - LG device type and model with issue: LG lse6335f range
 - LG devices connected (list): 

**Output of HA logs**
Paste the relavant output of the HA log here.

```
2022-02-26 09:29:39 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] No SmartThinQ devices found. Component setup aborted.
2022-02-26 09:32:17 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] No SmartThinQ devices found. Component setup aborted.
```

Country Code: US
Language Code: en
Account Type: LG Account

**Comment by ollo69** [comment]
Is your range registered in LG phone app using the same account?

**Comment by ollo69** [comment]
This error occurs when no device are found, both supported and unsupported, so it seems that after login the list of available device is empty. If you are sure that your range is properly registered in Mobile app using the same account used in this integration, you should enable debug and attach here the log file in this thread so that I can try to understand what is happening.

**Comment by SaintChimera** [comment]
Welp, before enabling debug logging i had my fiance confirm that the range was still working in the app. We discovered that she was using a Google account even though she has an LG account and thought she was using the LG account. We disassociated the range from the Google account and associated it with the LG account and everything worked as expected. I guess I'm happy that it was just a user error.

Sorry for the inconvenience and thank you for working with me!
